spec/support/shared_examples/register_api_endpoint.rb

```diff
@@ -2,8 +2,12 @@
 
 RSpec.shared_examples "a register API endpoint", openapi: false do |url, openapi: false|
   context "with a valid authentication token" do
+    let(:auth_token) do
+      super() || create(:authentication_token)
+    end
+
     let(:token) do
-      super() || create(:authentication_token).token
+      auth_token.token
```

Comment:
So that we can get at the `auth_token` and not just the `token` in the shared example.